message_sender/views.py

```diff
@@ -89,6 +90,18 @@ class InboundViewSet(viewsets.ModelViewSet):
                      'content', 'transport_name', 'transport_type',
                      'helper_metadata', 'created_at', 'updated_at',)
 
+    def create(self, request, *args, **kwargs):
+        expect = ["event_type", "in_reply_to"]
+        if set(expect).issubset(request.data.keys()):
+            if request.data['event_type'] == "close":
```

Comment:
I think it depends on the transport whether or not it keeps track and correctly marks the `reply_to`. This message was from the telnet transport, so not a transport that we use in production.

The `message_id` is from the current inbound message, not the outbound message.

I think a good strategy would be to look for the `reply_to`, and if it doesn't exist, then look for the last outbound message sent to `from`/`from_addr`, and if the from doesn't exist, or if we can't find a message, then we give up and rely on the timeout.